test/e2e/e2e_suite_test.go

```diff
@@ -40,7 +39,7 @@ func TestMain(m *testing.M) {
 	// Now that we know which Viper config (if any) was chosen,
 	// parse it and update those options which weren't already set via command line flags
 	// (which have higher priority).
-	if err := viperconfig.ViperizeFlags(*viperConfig, "e2e", flag.CommandLine); err != nil {
+	if err := viperizeFlags(*viperConfig, "e2e", flag.CommandLine); err != nil {
```

Comment:
this can be dropped https://github.com/ovn-org/ovn-kubernetes/pull/1561/commits/e1f22f001ac1bc25ca9c30d21a361da2b7122b47, I can't find my previous comment :)

Reply:
Done, also hardcoded the agnhost version instead of relying on framework as per your other comment